extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/OidcBuildStep.java

```diff
@@ -90,6 +97,18 @@ public SyntheticBeanBuildItem setup(
                 .done();
     }
 
+    @BuildStep(onlyIf = IsEnabled.class)
+    @Record(ExecutionTime.RUNTIME_INIT)
+    public void findSecurityEventObservers(
+            OidcRecorder recorder,
+            ValidationPhaseBuildItem validationPhase) {
```

Comment:
You should read the [ValidationPhaseBuildItem javadoc](https://github.com/quarkusio/quarkus/blob/master/extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/ValidationPhaseBuildItem.java#L15-L18) more carefully ;-). Your build step should produce or pretend that it produces a `ValidationErrorBuildItem`.